cryptovoxels/integration.js

```diff
@@ -292,12 +292,17 @@ async function loadBanner(space, network, format, style, beacon = true) {
   feature.set({'link': url});
 }
 
-// Call loadBanner here. Parameters are:
-// Space ID, Network, Format, Style, Enable Beacon (optional)
-feature.on('click', e =>{
-  sendOnClickMetric(89);
-});
-
-parcel.on('playerenter', e =>{
-  loadBanner(89, 'polygon', "square", "transparent");
-});
+if (SPACE >= 0 && NETWORK && FORMAT && STYLE && (BEACON !== undefined)) {
```

Comment:
What happens if the `SPACE` is negative?

Reply:
It would fail over and give the console.warn message, though I could make that a bit more explicit I guess